src/utils/console-writer.ts

```diff
@@ -0,0 +1,50 @@
+// @ts-ignore
+import columnify = require('columnify');
+import chalk from 'chalk';
+
+const COLORS = {
+  heading: chalk.rgb(224, 78, 58),
+};
+
+export default class ConsoleWriter {
+  columns: number;
+
+  constructor({ columns } = process.stdout) {
+    this.columns = columns || 80;
+  }
+
+  heading(heading: string) {
+    this.divider();
+    this.text(COLORS.heading(`${this.indent(2)}${heading}`));
+    this.divider();
+    this.line();
+  }
+
+  pairs(data: { [key: string]: string }) {
+    console.log(
+      columnify(data, {
+        dataTransform: (data: string) => {
+          return this.indent(2) + data;
```

Comment:
nit: use template literal for concatenation

Reply:
I mean, this is the difference of `${this.indent(2)}${data}` vs. the above. Does this really bother you?

Reply:
It's more about consistency than the difference above that I prefer :)
ref: https://github.com/scalvert/ember-cli-checkup/pull/50/files/8d3d160f44b8b90da09f9400634eb26d9e3fdbbc#diff-49e69fc097ef947f6d44ef1f35d72075R18

Reply:
Done!